models/Pokemon.go

```diff
@@ -0,0 +1,319 @@
+					FrontFemale      interface{} `json:"front_female"`
+					FrontShiny       string      `json:"front_shiny"`
+					FrontShinyFemale interface{} `json:"front_shiny_female"`
+				} `json:"omegaruby-alphasapphire"`
+				XY struct {
+					FrontDefault     string      `json:"front_default"`
+					FrontFemale      interface{} `json:"front_female"`
+					FrontShiny       string      `json:"front_shiny"`
+					FrontShinyFemale interface{} `json:"front_shiny_female"`
+				} `json:"x-y"`
+			} `json:"generation-vi"`
+			GenerationVii struct {
+				Icons struct {
+					FrontDefault string      `json:"front_default"`
+					FrontFemale  interface{} `json:"front_female"`
+				} `json:"icons"`
+				UltraSunUltraMoon struct {
+					FrontDefault     string      `json:"front_default"`
+					FrontFemale      interface{} `json:"front_female"`
+					FrontShiny       string      `json:"front_shiny"`
+					FrontShinyFemale interface{} `json:"front_shiny_female"`
+				} `json:"ultra-sun-ultra-moon"`
+			} `json:"generation-vii"`
+			GenerationViii struct {
+				Icons struct {
+					FrontDefault string      `json:"front_default"`
+					FrontFemale  interface{} `json:"front_female"`
+				} `json:"icons"`
+			} `json:"generation-viii"`
+		} `json:"versions"`
+	} `json:"sprites"`
+	Stats []struct {
+		BaseStat int `json:"base_stat"`
+		Effort   int `json:"effort"`
+		Stat     struct {
+			Name string `json:"name"`
+			URL  string `json:"url"`
+		} `json:"stat"`
+	} `json:"stats"`
+	Types []struct {
+		Slot int `json:"slot"`
+		Type struct {
+			Name string `json:"name"`
+			URL  string `json:"url"`
+		} `json:"type"`
+	} `json:"types"`
+	Weight int `json:"weight"`
+}
+
+// PokemonToCsvRow convert a pokemon structure to a string
+func PokemonToCsvRow(p Pokemon) string {
+	rowModel := [8]string{
+		strconv.Itoa(p.ID),
+		p.Name,
+		p.TypeOne,
+		strconv.Itoa(p.Total),
+		strconv.Itoa(p.HealthPoints),
+		strconv.Itoa(p.Attack),
+		strconv.Itoa(p.Defense),
+		strconv.Itoa(p.Speed),
+	}
+	return strings.Join(rowModel[:], ",") + "\n"
+}
+
+// PokemonFromString return a Pokemon from a string
+func PokemonFromString(csvRow string) Pokemon {
+	pokeRow := strings.Split(csvRow, ",")
+
+	id, _ := strconv.Atoi(pokeRow[0])
+	total, _ := strconv.Atoi(pokeRow[4])
+	healthPoints, _ := strconv.Atoi(pokeRow[5])
+	attack, _ := strconv.Atoi(pokeRow[6])
+	defense, _ := strconv.Atoi(pokeRow[7])
+	speed, _ := strconv.Atoi(pokeRow[10])
+
+	return Pokemon{
+		ID:           id,
+		Name:         pokeRow[1],
+		TypeOne:      pokeRow[2],
+		Total:        total,
+		HealthPoints: healthPoints,
+		Attack:       attack,
+		Defense:      defense,
+		Speed:        speed,
+	}
+
+}
+
+// StorePokemonInCsv store a pokemin in a file
+func StorePokemonInCsv(pokemon Pokemon) error {
+	poke := PokemonToCsvRow(pokemon)
+
+	readContent, err := helpers.ReadFile(fmt.Sprintf("%s/pokemon_v2.csv", constants.AssetsPath))
+
+	if err != nil {
+		return err
+	}
+
+	prevContent := strings.Join(readContent[:], "\n")
```

Comment:
```suggestion
	prevContent := strings.Join(readContent, "\n")
```
readContent[:] is the same as readContent.